packages/cli/src/create-app/runNpmInstall.cjs

```diff
@@ -0,0 +1,72 @@
+const { spawnSync } = require( 'child_process' );
+const ora = require( 'ora' );
+
+/**
+ * Runs npm install in the project directory with improved output handling.
+ *
+ * @param {string} projectDirPath - Path to the project directory.
+ * @throws {Error} - Throws an error if installation fails.
+ */
+const runNpmInstall = async ( projectDirPath ) => {
+	// Initialize spinner
+	const spinner = ora( {
+		text: 'Installing NPM dependencies',
+		color: 'cyan',
+	} );
```

Comment:
This was the change I made to get around the ERR_REQUIRE_ESM, not sure if it's the cause of my other issues.

```suggestion
const { spawnSync } = require( 'child_process' );

/**
 * Runs npm install in the project directory with improved output handling.
 *
 * @param {string} projectDirPath - Path to the project directory.
 * @throws {Error} - Throws an error if installation fails.
 */
const runNpmInstall = async ( projectDirPath ) => {
	// Import ora dynamically
	const { default: ora } = await import( 'ora' );

	// Initialize spinner
	const spinner = ora( {
		text: 'Installing NPM dependencies',
		color: 'cyan',
	} );
```